prc/ecdata/specialized/ecpairing.tex

```diff
@@ -42,7 +42,18 @@
 \begin{description}
     \item[\underline{Row $n^°(i+4)$:}]
           \[
-              \callToWellFormedCoordinates{4}{\locBXImHi}{\locBXImLo}{\locBXReHi}{\locBXReLo}{\locBYImHi}{\locBYImLo}{\locBYReHi}{\locBYReLo}
+              \callToWellFormedCoordinates {
+                anchorRow  = i                      ,
+                relOffset  = 4                      ,
+                xImHi      = \locBXImHi             ,               
+                xImLo      = \locBXImLo             ,               
+                xReHi      = \locBXReHi             ,            
+                xReLo      = \locBXReLo             ,           
+                yImHi      = \locBYImHi             ,         
+                yImLo      = \locBYImLo             ,         
+                yReHi      = \locBYReHi             ,         
+                yReLo      = \locBYReLo             ,           
+            }
           \]
 
           \[
```

Comment:
This space
```latex
          \]

          \[
```
should be removed. There are better ways to create separation if this is the goal.